src/main/java/seedu/address/logic/commands/ConfirmationCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Confirms whether or not to execute a command.
+ */
+public class ConfirmationCommand extends Command {
+    public static final String MESSAGE_CONFIRMATION = "(yes/no)";
+    public static final String MESSAGE_CONFIRMATION_DELETE = "Delete %1$d? " + MESSAGE_CONFIRMATION;
+    public static final String MESSAGE_CONFIRMATION_EDIT = "Edit %1$d? " + MESSAGE_CONFIRMATION;
+    public static final String MESSAGE_CONFIRMATION_CLEAR = "Clear list? " + MESSAGE_CONFIRMATION;
+    public static final String ACCEPT_COMMAND_1 = "yes";
+    public static final String ACCEPT_COMMAND_2 = "y";
+    public static final String REJECT_COMMAND_1 = "no";
+    public static final String REJECT_COMMAND_2 = "n";
```

Comment:
Perhaps these could be renamed to `ACCEPT_COMMAND_FULL`, `ACCEPT_COMMAND_SHORT`, `REJECT_COMMAND_FULL` and `REJECT_COMMAND_SHORT`?